### PR TITLE
refactor(ui5-tag): wrap text by default

### PR DESCRIPTION
**Pull request by LidiyaGeorgieva**
The text of `ui5-tag` now wraps by default.

BREAKING CHANGE: `wrapping-type` property default value has changed from `None` to `Normal`. Before:
```html
<ui5-tag>In Process</ui5-tag><!-- would truncate the text if there is not enough space -->
```

Now:
```html
<ui5-tag>In Process</ui5-tag><!-- would let the text wrap if there is not enough space -->
```

Related to https://github.com/SAP/ui5-webcomponents/issues/8461

### CODE REVIEW

**Comment by BorisDafov** [comment]
I approve the PR from UA perspective

File: packages/main/test/pages/Tag.html
Comment (GerganaKremenska) [inline packages/main/test/pages/Tag.html:1]: remove all occurences of wrapping-type="Normal" as this is the default and it is applied by default.

File: packages/main/src/themes/Tag.css [hunk 1]
```diff
@@ -82,15 +82,20 @@
...
+	height: var(--_ui5-tag-icon-height);
 	color: inherit;
 	pointer-events: none;
+	align-self: flex-start;
 }
 
-:host([wrapping-type="Normal"]) [ui5-icon],
-:host([wrapping-type="Normal"]) ::slotted([ui5-icon]) {
-	align-self: flex-start;
+:host([_is-tag-design]) [ui5-icon],
```
Comment (GerganaKremenska) [inline packages/main/src/themes/Tag.css:91]: _is-tag-design is regarding design="Set3", as it is deleted with this pull https://github.com/SAP/ui5-webcomponents/pull/9115/files#diff-9dc0bd3858a768f765383f49db1243e566677dae75941c9e40048637c57e0b7d

You can delete this selector regarding"_is-tag-design"

File: packages/main/src/themes/base/Tag-parameters.css [hunk 2]
```diff
@@ -5,7 +5,7 @@
 	--_ui5-tag-padding-inline: 0.3125em;
 	--_ui5-tag-padding-inline-icon-only: 0.1875rem;
 	--_ui5-tag-text-transform: uppercase;
-	--_ui5-tag-icon-height: 0.75em;
+	--_ui5-tag-icon-height: calc(var(--_ui5-tag-height) - 2px); /* remove width of the border since tag root is with box-sizing:border-box*/
```
Comment (GerganaKremenska) [inline packages/main/src/themes/base/Tag-parameters.css:8]: change 2px to 0.125rem

**Review by GerganaKremenska — CHANGES_REQUESTED** [review]

File: packages/main/src/themes/Tag.css [hunk 3]
```diff
@@ -82,15 +82,20 @@
...
+:host([_is-tag-design]) ::slotted([ui5-icon]) {
+	height: calc(var(--_ui5-tag-height) - 2px); /* remove width of the border since tag root is with box-sizing:border-box*/
```
Comment (ilhan007) [inline packages/main/src/themes/Tag.css:93]: you have the calculation in a parameter : `--_ui5-tag-icon-height` in base/Tag-parameters.css, or it does not work somehow?

**Pull request merged**